packages/api/src/models/player.ts

```diff
@@ -73,16 +79,45 @@ export class PlayerModel {
 
   public async update(player: DbPlayer): Promise<DbPlayer> {
     const { username } = player
-    const exists = await this.repository.getOne({ username })
-
-    if (!exists) {
-      throw new Error(`Player does not exist (name: ${player.username})`)
-    }
+    // const exists = await this.repository.getOne({ username })
```

Comment:
Why remove this?

Reply:
To avoid making an extra call. This method calls the ´repository.updateOne´, which checks if the resource exists before updating it.